src/harness/testcases/WINNF_FT_S_DRG_testcase.py

```diff
@@ -269,8 +272,161 @@ def test_WINNF_FT_S_DRG_5(self):
         }]
     }
     response = self._sas.Heartbeat(request)['heartbeatResponse'][0]
-    
+
     # Check the heartbeat response
     self.assertEqual(response['response']['responseCode'], 103)
     del request, response
 
+  def generate_DRG_6_default_config(self, filename):
+    """Generates the WinnForum configuration for DRG.6."""
+
+    # Load device info
+    device_a = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_a.json')))
+    device_c = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_c.json')))
+    device_b = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_b.json')))
+
+    # Load grant.json
+    grant_a = json.load(
+        open(os.path.join('testcases', 'testdata', 'grant_0.json')))
+    grant_c = json.load(
+        open(os.path.join('testcases', 'testdata', 'grant_0.json')))
+    grant_b = json.load(
+        open(os.path.join('testcases', 'testdata', 'grant_0.json')))
+
+    # Heartbeat requests (where testcode will insert cbsdId and grantId)
+    heartbeat_a = {'operationState': 'GRANTED'}
+    heartbeat_c = {'operationState': 'GRANTED'}
+    heartbeat_b = {'operationState': 'GRANTED'}
+
+    # Deregistration requests (where testcode will fill in cbsdId)
+    deregister_a = {}
+    # device_c's CBSD ID will be removed due to the 'REMOVE' keyword.
+    deregister_c = {'cbsdId': 'REMOVE'}
+    deregister_b = {}
```

Comment:
Perhaps we can make one of these deregistration requests to contain an invalid cbsdId value.

Reply:
Good point! Done.